<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>895</width>
    <height>654</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>840</width>
    <height>620</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="appresources.qrc">
    <normaloff>:/Icons/Icons/barchart.png</normaloff>:/Icons/Icons/barchart.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTreeView" name="treeView">
        <property name="maximumSize">
         <size>
          <width>460</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="alternatingRowColors">
         <bool>false</bool>
        </property>
        <property name="rootIsDecorated">
         <bool>true</bool>
        </property>
        <attribute name="headerCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="headerHighlightSections">
         <bool>true</bool>
        </attribute>
        <attribute name="headerMinimumSectionSize">
         <number>57</number>
        </attribute>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="resetPlotButton">
          <property name="text">
           <string>Reset Plot</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveJPGButton">
          <property name="text">
           <string>Save JPG</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stopPlotButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Stop Plot</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbox_axis_x_pointes">
          <item>
           <property name="text">
            <string>10 seconds</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>20 seconds</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>30 seconds</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>40 seconds</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>50 seconds</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1 minute</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2 minutes</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3 minutes</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4 minutes</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5 minutes</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>538</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCustomPlot" name="plot" native="true"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>895</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelo">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHow_to_use"/>
   </widget>
   <widget class="QMenu" name="menuConfig">
    <property name="title">
     <string>Setup</string>
    </property>
    <addaction name="actionGeneral"/>
    <addaction name="actionCommLinks"/>
   </widget>
   <addaction name="menuConfig"/>
   <addaction name="menuHelo"/>
  </widget>
  <action name="actionHow_to_use">
   <property name="text">
    <string>How to use </string>
   </property>
  </action>
  <action name="actionCommLinks">
   <property name="text">
    <string>Comm Links</string>
   </property>
  </action>
  <action name="actionUDP">
   <property name="text">
    <string>UDP</string>
   </property>
  </action>
  <action name="actionGeneral">
   <property name="text">
    <string>General</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="appresources.qrc"/>
 </resources>
 <connections/>
</ui>
